getting-started/best-practices.mdx

```diff
@@ -92,7 +92,7 @@ See our documentation on [Loading Assets](/runtimes/loading-assets). This featur
 
 ### Caching your .riv
 
-If you’re going to use the same Rive file in multiple places throughout your page or app, it’s possible to [cache the .riv](/runtimes/caching-a-rive-file) so that your users only download it once.
```

Comment:
> i would change this to not just say "download it", because downloading an asset is only relevant for the web runtime really.

> The real benefit is that you don't have to decode and parse the .riv file multiple times. Spawning an artboard instance from an already decoded .riv file is much faster than first decoding the .riv file and then creating an artboard instance.